docs/get-started.md

```diff
@@ -98,8 +98,15 @@ var client = new WebTorrent()
 var magnetURI = 'magnet: ...'
 
 client.add(magnetURI, { path: '/path/to/folder' }, function (torrent) {
+  var isDone = false
+
   torrent.on('done', function () {
-    console.log('torrent download finished')
+    console.log('\ntorrent download finished')
+    isDone = true
+  })
+
+  torrent.on('download', function (bytes) {
+    if (!isDone) process.stdout.write(`\rtotal: ${torrent.length}  progress: ${(client.progress * 100).toFixed(2)}%  ${Math.round(client.downloadSpeed / 1024).toFixed(1)} KB/s  downloading...`)
```

Comment:
```suggestion
    process.stdout.write(`\r(${Math.round(torrent.downloaded / Math.pow(1024,2))}/${Math.round(torrent.length / Math.pow(1024,2))}MB) [${Math.round(torrent.downloadSpeed / 1024).toFixed(1)}KB/s][${Math.round(torrent.uploadSpeed / 1024).toFixed(1)}KB/s] (${torrent.numPeers}) [${Math.floor((torrent.timeRemaining / (60 * 1000)) % 60)}:${Math.floor((torrent.timeRemaining / 1000) % 60)}] [${(torrent.progress * 100).toFixed(2)}%]`)  })
```

Reply:
I had one other pull request that printed out more info, but I also noticed that unless if I use 

```
const go_beginning_of_line = "\033[G"
const clear_till_end_of_line = "\033[K"
```
to deal with the output, then using the `\r`, if the line is wider than the terminal's width, then the line would wrap to the next line and `\r` only goes back to the position one of that next line. So the screen will have many lines of output in this case.  So that's why I kept the output short, just to show some status.  Also if I use those ANSI escape sequence, it may not work if the terminal is not compatible with it and I really should use a `curses` library... but then it probably defeats the purpose of the author showing a simple program to show how it can download all the files for a torrent and save them in a directory.

Reply:
true... I really should use `torrent.downloadSpeed` and I will correct it in the next minute.  Just this case it is one torrent so the client and torrent have the same speed.